error.go

```diff
@@ -91,15 +91,37 @@ func (e *LibraryError) Error() string {
 
 // LocalError represents an error caused by the EasyPost library itself, such as validation or JSON serialization issues.
 type LocalError struct {
-	LibraryError
+	LibraryError // subtype of LibraryError
+}
+
+// Unwrap returns the underlying LibraryError object.
+func (e *LocalError) Unwrap() error {
+	return &e.LibraryError
 }
 
 // EndOfPaginationError is raised when there are no more pages to retrieve.
-var EndOfPaginationError = &LocalError{LibraryError{Message: NoPagesLeftToRetrieve}}
+type EndOfPaginationError struct {
+	LocalError // subtype of LocalError
+}
+
+// Unwrap returns the underlying LocalError object.
+func (e *EndOfPaginationError) Unwrap() error {
+	return &e.LocalError
+}
+
+// newEndOfPaginationError returns a new EndOfPaginationError object.
+func newEndOfPaginationError() *EndOfPaginationError {
+	return &EndOfPaginationError{LocalError{LibraryError{Message: NoPagesLeftToRetrieve}}}
+}
 
 // FilteringError is raised when there is an issue while running a filtering operation.
 type FilteringError struct {
-	LocalError
+	LocalError // subtype of LocalError
```

Comment:
```suggestion
	*LocalError // subtype of LocalError
```

update all errors that have an error subtype to be a pointer to the error instead.

Reply:
not convinced this is entirely necessary

Reply:
This seems to break things in my IDE, and I don't see any documentation about Go "inheritance" that suggests this.